tensorvis/tensorvisbase/src/processors/tensorfield2dasrgba.cpp

```diff
@@ -27,30 +27,35 @@
  *
  *********************************************************************************/
 
-#include <inviwo/tensorvisbase/processors/tensorfieldtorgba.h>
+#include <inviwo/tensorvisbase/processors/tensorfield2dasrgba.h>
 #include <inviwo/tensorvisbase/algorithm/tensorfieldsampling.h>
+#include <inviwo/core/util/imagesampler.h>
+#include <inviwo/tensorvisbase/tensorvisbasemodule.h>
 
 namespace inviwo {
 
 // The Class Identifier has to be globally unique. Use a reverse DNS naming scheme
-const ProcessorInfo TensorFieldToRGBA::processorInfo_{
-    "org.inviwo.Tensor Field To RGBA",  // Class identifier
-    "TensorFieldToRGBA",                // Display name
-    "Tensor visualization",             // Category
-    CodeState::Experimental,            // Code state
-    Tags::GL,                           // Tags
+const ProcessorInfo TensorField2DAsRGBA::processorInfo_{
+    "org.inviwo.TensorField2DAsRGBA",  // Class identifier
+    "Tensor Field 2D As RGBA",         // Display name
+    "Tensor visualization",            // Category
+    CodeState::Experimental,           // Code state
+    tag::OpenTensorVis | Tag::GL,      // Tags
```

Comment:
note that the namespace of `tag::OpenTensorVis` has a lowercase `T`

Reply:
that was on purpose for two reasons: (1) naming convention within Inviwo as namespaces are always lowercase (2) distinguish it from the default tags.